src/ontology/mappings/d3fend-cco.ttl

```diff
@@ -87,6 +87,61 @@ d3f:Technique rdfs:subClassOf cco:DirectiveInformationContentEntity.
 
 d3f:InformationContentEntity owl:equivalentClass cco:InformationContentEntity.
 
+d3f:Log rdfs:subClassOf cco:RepresentationalInformationContentEntity .
+
+# An example cco gives of a representational ICE is a transcript, which seems similar to a log.
+
+d3f:Sensor owl:equivalentClass cco:Sensor .
+
+d3f:Artifact rdfs:subClassOf <http://purl.obolibrary.org/obo/BFO_0000002> .
+
+#obo:BFO_0000002 aka "continuant".
+#like Digital Artifact, it can't be a subclass of Material Artifact or Object for the same reason. 
+#Though the definition they give of Artifact is cco Material Artifact, which can't be right for this reason.
+
+d3f:Capability owl:equivalentClass cco:AgentCapability .
+
+#they say it's defined by dbpedia and linked a definition for capability in 'systems engineering', 
+#but I didn't find anything in the definition that made it seem like the definition wasn't applicable
+#to capability in general. 
+
+d3f:DigitalObject rdfs:subClassOf <http://purl.obolibrary.org/obo/BFO_0000002> .
+
+#obo:BFO_0000002 aka "continuant".
+#If this is the place Digital Artifact needs to go, it's also where Digital Object should go, since Digital Artifact is a subclass of Digital Object.
+
+d3f:Monitoring owl:equivalentClass cco:ActOfObservation .
+
+#They define monitoring as an act of observing something. They also add that it might involve keeping a record of something.
+
+d3f:Procedure rdfs:subClassOf cco:DirectiveInformationContentEntity .
+
+#This seems like a Plan. They don't give a definition of Procedure, and the only subclass is Use Case Procedure.
+
+d3f:Vulnerability rdfs:subClassOf <http://purl.obolibrary.org/obo/BFO_0000016> .
+
+#obo:BFO_0000016 aka "disposition".
+#Like 'Weakness', Vulnerability seems like a disposition.
+
+d3f:AnalysisOfAlternatives rdfs:subClassOf cco:Act .
+
+#This might further be a planned act. They don't define this term, so it's difficult to tell.
+
+d3f:Assessment rdfs:subClassOf cco:ActOfAppraisal .
+
+#They define it as classifying something according to its worth.
+
+d3f:Command rdfs:subClassOf cco:DirectiveInformationContentEntity .
+
+#Command is defined as being a directive to a computer program.
```

Comment:
But command is a digital event, so it seems to me it should be an occurrent, not a continuant.

Reply:
They also say command is digital information. It looks like they're not sure what it's supposed to be. I'll delete this one.